prime-router/src/main/kotlin/Schema.kt

```diff
@@ -41,16 +41,16 @@ data class Schema(
         val missing = mutableSetOf<String>()
 
         toSchema.elements.forEach { toElement ->
-            findMatchingElement(toElement)?.let {
-                useDirectly[toElement.name] = it
-                return@forEach
-            }
             findMatchingValueSet(toElement)?.let {
```

Comment:
@RickHawesUSDS  - please check this.  I swapped the order of the checks here, to give the 'useValueSet' higher priority.  Before, that option was unreachable code - so valueSets were never being properly processed.